mantidproject/mantid
# Indirect Improve Release Notes

**Description of work.**
This PR improves the indirect release notes by adding several images.
This PR also fixes a bug on Sqw where you select the TOSCA instrument and click `Run`. This would cause an unexpected error as the TOSCA instrument doesn't have an EFixed.

**To test:**
Read through the documentation and check the new images.

1.`Interfaces`->`Indirect`->`Data Reduction`->`Sqw` tab
2. Select `TOSCA` as the instrument.
3. Load the file below.
4. Q-Low Q-Width Q-High: 0.5-0.05-1.8
5. Click `Run`. A meaningful error message should appear.

**Data Files**
[irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/2977929/irs26176_graphite002_red.zip)

Fixes #24922 #24225 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
